packages/astro-font/utils.ts

```diff
@@ -5,30 +5,34 @@ import { Buffer } from 'node:buffer'
 import { getFallbackMetricsFromFontFile } from './font.ts'
 import { pickFontFileForFallbackGeneration } from './fallback.ts'
 
-interface Record {
-  [property: string]: string
-}
+type GlobalValues = "inherit" | "initial" | "revert" | "revert-layer" | "unset"
+
+type FontWeight = 100 | 200 | 300 | 400 | 500 | 600 | 700 | 800 | 900
 
 interface Source {
   path: string
-  css?: Record
-  style: string
+  css?: Record<string, string>
+  // https://developer.mozilla.org/en-US/docs/Web/CSS/font-style
+  style: "normal" | "italic" | "oblique" | `oblique ${number}deg` | GlobalValues | {}
```

Comment:
Alright so here is a little TS trickery:
- if you set your type to `'a' | 'b'`, your IDE will show you both but won't let you input any string
- if you set your type to `'a' | 'b' | string`, your IDE will let you input any string but a and b autocompletion is lost
- if you set your types to `'a' | 'b' | {}`, your IDE will show you both but will let you input any string

I did this trick for several properties below as well